app/views/dashboard/advisers/_advisers_table.html.erb

```diff
@@ -9,18 +9,29 @@
       </tr>
     </thead>
     <tbody>
-      <% advisers.each do |adviser| %>
-        <tr class="t-advisers-table-row">
-          <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
-          <td class="t-adviser-name"><%= adviser.name %></td>
-          <td>
-            <ul class="l-comma-separated-list">
-              <% adviser.qualifications.each do |qualification| %>
-                <li class="t-adviser-qualification"><%= qualification.name %></li>
-              <% end %>
-            </ul>
+      <% if firm.advisers.present? %>
+        <% firm.advisers.each do |adviser| %>
+          <tr class="t-advisers-table-row">
+            <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
+            <td class="t-adviser-name"><%= adviser.name %></td>
+            <td>
+              <ul class="l-comma-separated-list">
```

Comment:
Kind of. The way I thought about it was that `l-comma-separated-list` 'lays out' a set of list items, i.e. it lays them out horizontally rather than vertically.

But what I'm understanding from you is that a layout — actually I still don't understand. I'm finding it hard to reconcile 'layout of components' with things like `l-constrained`.

So I now know that a `ul` of `li`s is not a layout. If I had a `div` of `button`s that I wanted to lay out side-by-side. Would that be a layout or a component?

Not trying to be difficult! I just know it's something that's going to pay off for me to understand, so trying to do so. Previously I just used used `l-` classes for things that do high level laying out on a page, like `l-2col-even` for instance — but this seems like another important use case to understand.